bip-p2qrh.mediawiki

```diff
@@ -68,9 +64,17 @@ The proposal above also leaves a gap in case it makes sense to use version 2, or
 
 The name P2QRH means Pay to Quantum Resistant Hash. This is similar to how P2PKH, as defined in [https://github.com/bitcoin/bips/blob/master/bip-0016.mediawiki#specification BIP-16].
```

Comment:
I know this isn't a change you made, but we should probably give the unabbreviated version of P2QRH the first time it is mentioned.  I believe that is in the Motivation section.  That way the reader doesn't have to guess until they get to this point.